x/profiles/simulation/operations_profile.go

```diff
@@ -29,13 +29,13 @@ func SimulateMsgSaveProfile(k keeper.Keeper, ak auth.AccountKeeper) sim.Operatio
 			return sim.NoOpMsg(types.ModuleName), nil, nil
 		}
 
-		var profilePic, coverPic = "", ""
+		var profilePic, coverPic = RandomProfilePic(r), RandomProfileCover(r)
```

Comment:
Why was this changed? Shouldn't we also test if the profile pic and cover pics are empty?

Reply:
A particular seed raised a bug inside this test.
Basically if there was no pictures it would have assigned the empty string leading to an error when later inside
the profile validation.
You were right to point me this because I found a better solution which is set the two vars to `nil` by default instead of an empty string, which is basically what happens if someone doesn't share any profile image.